### PR TITLE
Refactor/recipe utils test file errors

### PR DESCRIPTION
**Pull request by fernandorobles97**
What I did:
- Deleted unnecessary tests.
- Fixed expect statement to equal a single string instead of an array of strings.
- Passed additional argument in calculateRecipeCost invocation so it works properly.
- Create a describe block for every function that didn't have one.

Did this break anything?
- [x] No
- [ ]  Yes

Type of change
- [ ] New feature (non-breaking change which adds functionality)
- [x]  Bug fix (non-breaking change which fixes an issue)
- [x]  Refactor(DRY-ing up/ reorganizing code, etc.)
- [x]  Super small fix (Corrected a typo, removed a comment, etc.)
- [ ]  Skip all the other stuff and briefly explain the fix.

Checklist:
- [x]  If this code needs to be tested, all tests are passing
- [x]  The code runs locally
- [x]  There are comments where clarification/ organization is needed
- [x]  The code is DRY. If it's not, I tried my best
- [x]  I reviewed my code before pushing

What's next?
Finishing iteration 2 user stories.

### CODE REVIEW

**Review by hvohr — APPROVED** [review]
This is super clean and DRY testing, love it

File: test/RecipeUtils-test.js [hunk 1]
```diff
@@ -5,16 +5,18 @@ import ingredientsData from '../src/data/ingredients-sample.js';
 import { recipesToCook, saveRecipe } from '../src/userUtils';
 
 
-describe('RecipeUtils', () => {
+describe('recipesFromTag', () => {
```
Comment (andreasorensen) [inline test/RecipeUtils-test.js:8]: I like having each function in it's own describe block! That looks clean.

File: test/RecipeUtils-test.js [hunk 2]
```diff
@@ -41,30 +45,33 @@ describe('RecipeUtils', () => {
...
   it ('Should filter recipes when multiple tags are applied', () => {
     const expected = ['Thai Chicken Tenders with Broiled Pineapple Slaw', 'Maple Dijon Apple Cider Grilled Pork Chops']
     const filtered = recipesFromTag(recipeData, ['lunch', 'main dish'])
     expect(filtered.length).to.equal(2)
     const names = filtered.map(f => f.name)
     expect(names).to.have.members(expected)
   });
+});
+
+describe('recipesFromName', () => {
```
Comment (Denzel-Turambi) [inline test/RecipeUtils-test.js:58]: Good call on making a new describe block for this function!

**Review by Denzel-Turambi — APPROVED** [review]

File: test/RecipeUtils-test.js [hunk 3]
```diff
@@ -96,26 +103,10 @@ describe('findIngredientNames', () => {
...
-      'semi sweet chips',
-      'unsalted butter',
-      'vanilla' ]);
-  });
-
-  it.skip('should return message if no recipe is found', () => {
-    const recipeName = 'Not a recipe';
-    const ingredients = findIngredientNames(recipeData, ingredientsData,recipeName);
-    expect(ingredients).to.equal("Sorry, we don't have that recipe.");
+    expect(ingredients).to.equal('wheat flour, bicarbonate of soda, eggs, sucrose, instant vanilla pudding, brown sugar, salt, fine sea salt, semi sweet chips, unsalted butter, vanilla');
```
Comment (andreasorensen) [inline test/RecipeUtils-test.js:109]: Thanks for updating that!

**Review by andreasorensen — APPROVED** [review]
Thanks for refactoring all of these! Good catch on the double tests.

**Pull request merged**